src/main/scala/com/ebiznext/comet/job/ingest/IngestionJob.scala

```diff
@@ -813,6 +814,7 @@ trait IngestionJob extends SparkJob {
     withScriptFieldsDF: DataFrame,
     mergeOptions: MergeOptions
   ): (DataFrame, Option[List[String]]) = {
+    val incomingSchema = BigQueryUtils.normalizeSchema(schema.finalSparkSchema(schemaHandler))
```

Comment:
You are calling BQ specific code within the parquet path. Is this normal ?
```suggestion
    val incomingSchema = schema.finalSparkSchema(schemaHandler)
```

Reply:
You're right!